nixops_aws/resources/ec2_rds_dbinstance.py

```diff
@@ -28,8 +30,8 @@ class EC2RDSDbInstanceDefinition(nixops.resources.ResourceDefinition):
 
     config: Ec2RdsDbinstanceOptions
     subnet_group: Optional[str]
-    vpc_security_groups: Optional[Sequence[str]] = None
-    rds_dbinstance_security_groups: Optional[Sequence[str]] = None
+    vpc_security_groups: Optional[Sequence[str]] = []
```

Comment:
So, is it still relevant to keep `Optional` ? Same for the line below.

Reply:
It is still relevant of course, the 'vpc_security_groups' can be omitted since it's optional but by default it would have the empty array value.